common/agents/tasks/scripts/forge-review-submit-github.sh

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env bash
+# Submit a forge review to GitHub
+#
+# Usage: forge-review-submit-github.sh [--dry-run] OWNER REPO PR_NUMBER [REVIEW_FILE]
+#
+# Options:
+#   --dry-run    Validate the review file without submitting
+#
+# Arguments:
+#   OWNER        Repository owner
+#   REPO         Repository name
+#   PR_NUMBER    Pull request number
+#   REVIEW_FILE  Path to JSONL file (default: .git/review.jsonl)
+#
+# Environment:
+#   GH_TOKEN or GITHUB_TOKEN - GitHub authentication token (optional if gh is configured)
+#
+# The JSONL file must be created with forge-review-start.sh and contain:
+#   Line 1: {"body": "Review body with attribution..."}
+#   Line 2+: {"path": "src/lib.rs", "line": 42, "body": "..."}
+#
+# On success, removes the review file. On failure, preserves it.
+
+set -euo pipefail
+
+usage() {
+    echo "Usage: $0 [--dry-run] OWNER REPO PR_NUMBER [REVIEW_FILE]" >&2
+    echo "" >&2
+    echo "Options:" >&2
+    echo "  --dry-run    Validate the review file without submitting" >&2
+    echo "" >&2
+    echo "REVIEW_FILE must be created with forge-review-start.sh first." >&2
+    exit 1
+}
+
+DRY_RUN=0
+if [[ "${1:-}" == "--dry-run" ]]; then
+    DRY_RUN=1
+    shift
+fi
+
+if [[ $# -lt 3 ]]; then
+    usage
+fi
+
+OWNER="$1"
+REPO="$2"
+PR_NUMBER="$3"
+REVIEW_FILE="${4:-.git/review.jsonl}"
+
+if [[ ! -f "$REVIEW_FILE" ]]; then
+    echo "Error: Review file not found: $REVIEW_FILE" >&2
+    exit 1
+fi
+
+# Parse JSONL: extract metadata and comments separately
+if ! ALL_ENTRIES=$(jq -s '. // []' < "$REVIEW_FILE" 2>&1); then
```

Comment:
AI: (low) The JSONL parsing logic (lines 57-86) is duplicated across all three submit scripts. Consider extracting this into a shared helper function or script to reduce maintenance burden when the format evolves.